spec/application.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
     },
     version: {
       type: 'string',
-      pattern: '^v2$',
+      pattern: '^(v1|v2)$',
```

Comment:
no, this will allow v1 applications in to the pipeline as valid, but they are not correctly formed for the rest of the spec of process them.

We could make it map v1 > v2, but that's a whole lot of work. Ben said it's fine to completely deprecate v1 applications as long as they don't show up in the app in a way which they can't be dismissed. This way we will never see them again

Reply:
Oh right, i think i was just testing something out when i did this

Reply:
All good